src/components/Table/sgds-table.scss

```diff
@@ -1,3 +1,26 @@
+:host{
+  //table
+  --table-bg: transparent;
+  --table-accent-bg: transparent;
+  --table-striped-color: var(--sgds-gray-600);
```

Comment:
hmm..should there be a variable for non-striped color as well? What about the tableHeaders part (bg & text) and sort icon?